Resources/public/templates/usermenuitemfireevent.hbt

```diff
@@ -1 +1 @@
-<a href="#" class="ez-user-menu-item">{{ title }}</a>
+<a href="#" class="ez-user-menu-item" menu-title="{{ title }}">{{ title }}</a>
```

Comment:
`menu-title` is not a valid attribute name. other than that, that's exactly the same case as in https://github.com/ezsystems/PlatformUIBundle/pull/653/files#r72211802, we should expose in the markup the event name so that you can identify the link.